install.sh

```diff
@@ -33,6 +36,29 @@ MAGENTA='\e[1;35m';
 RESET='\e[0m'
 
 echo -e "${GREEN}Welcome to Chromebrew!${RESET}\n"
+
+# disallow non Chrome OS devices
+if [ -z "${CHROMEOS_RELEASE_VERSION}" ]; then
+  echo -e "${RED}Only Chrome OS developer shell is supported by Chromebrew :/${RESET}"
+  exit 1
+fi
```

Comment:
This could end up locking out a lot of users if `CHROMEOS_RELEASE_VERSION` is not set.  Could we make a better assumption?

Reply:
This variable is set by `source /etc/lsb_release`, it is set unless non chrome os system or file not exist